docs/source/library/python/mix_ins/label_data.rst

```diff
@@ -0,0 +1,18 @@
+:mod:`Label Data` --- Label Data Mix-ins
+========================================
+
+The :mod:`ale.drivers.label_isis` module
```

Comment:
Should this be more generic? Something similar to the little description in the supplemental data mix-ins?